vspace/src/main/java/edu/asu/diging/vspace/core/model/IdPrefix.java

```diff
@@ -0,0 +1,17 @@
+package edu.asu.diging.vspace.core.model;
+
+public enum IdPrefix {

```

Comment:
these should also be used in the ID annotation of the domain classes, to avoid changing one but not the other.